src/TuringTutorials.jl

```diff
@@ -7,117 +7,120 @@ cssfile = joinpath(@__DIR__, "..", "templates", "skeleton_css.css")
-    catch
+function weave_folder(
+    folder, build_list=(:script,:html,:pdf,:github,:notebook);
+    ext = r"^\.[Jj]md", kwargs...
+)
+    for file in readdir(joinpath(repo_directory,"tutorials",folder))
+        try
+            # HACK: just making sure that we're never going to 
+            if occursin(ext, splitext(file)[2])
+                println("Building $(joinpath(folder,file))")
+                weave_file(folder, file, build_list; kwargs...)
+            else
+                println("Skipping $(joinpath(folder,file))")
+            end
+        catch ex
+            @warn "Weave failed" ex
+        end
     end
-  end
 end
 
 function tutorial_footer(folder=nothing, file=nothing; remove_homedir=true)
     display("text/markdown", """
-    ## Appendix
-     This tutorial is part of the TuringTutorials.jl repository, found at: <https://github.com/TuringLang/TuringTutorials.jl>..
-    """)
+        ## Appendix
+         This tutorial is part of the TuringTutorials.jl repository, found at: <https://github.com/TuringLang/TuringTutorials.jl>.
+        """)
     if folder !== nothing && file !== nothing
         display("text/markdown", """
-        To locally run this tutorial, do the following commands:
-        ```
-        using TuringTutorials
-        TuringTutorials.weave_file("$folder","$file")
-        ```
-        """)
+            To locally run this tutorial, do the following commands:
+            ```{julia; eval = false}
+            using TuringTutorials
+            TuringTutorials.weave_file("$folder", "$file")
+            ```
+            """)
     end
     display("text/markdown", "Computer Information:")
     vinfo = sprint(InteractiveUtils.versioninfo)
     display("text/markdown",  """
-    ```
-    $(vinfo)
-    ```
-    """)
+        ```
+        $(vinfo)
+        ```
+        """)
 
     ctx = Pkg.API.Context()
-    pkgs = Pkg.Display.status(Pkg.API.Context(), use_as_api=true);
+
+    pkg_status = let io = IOBuffer()
+        Pkg.status(Pkg.API.Context(); io = io)
+        String(take!(io))
+    end
     projfile = ctx.env.project_file
     remove_homedir && (projfile = replace(projfile, homedir() => "~"))
 
     display("text/markdown","""
-    Package Information:
-    """)
-
-    md = ""
-    md *= "```\nStatus `$(projfile)`\n"
+        Package Information:
+        """)
 
-    for pkg in pkgs
-        if !isnothing(pkg.old) && pkg.old.ver !== nothing
-          md *= "[$(string(pkg.uuid))] $(string(pkg.name)) $(string(pkg.old.ver))\n"
-        else
-          md *= "[$(string(pkg.uuid))] $(string(pkg.name))\n"
-        end
-    end
-    md *= "```"
+    md = "```\n$(pkg_status)\n```"
     display("text/markdown", md)
 end
 
 function open_notebooks()
-  Base.eval(Main, Meta.parse("import IJulia"))
-  path = joinpath(repo_directory,"notebook")
-  IJulia.notebook(;dir=path)
+    Base.eval(Main, Meta.parse("import IJulia"))
+    path = joinpath(repo_directory,"notebook")
+    IJulia.notebook(;dir=path)
 end
```

Comment:
I'm not sure if this is more convenient than just defining a function/constant that returns the notebook directory and tell users to run:
```julia
using IJulia, TuringTutorials
notebook(; dir=TuringTutorials.notebookdir())
```
That would be more explicit and avoid the nasty `eval`...

Reply:
I agree with David on this one. Having `TuringTutorials.open_notebooks()` seems like an antipattern.

Reply:
I agree with you guys :+1: Didn't give it any thought, but just copy-pasted.

Reply:
Removed it :+1: